cloudify_agent/api/pm/detach.py

```diff
@@ -64,11 +64,62 @@ def before_self_stop(self):
         self._runner.run(self.create_disable_cron_script())
         super(DetachedDaemon, self).before_self_stop()
 
+    def _clean_queue(self, client):
```

Comment:
```suggestion
    def _delete_queue(self, client, queue_name):
```
it's not _cleaning_ the queue, it's deleting it